infrastructure/ansible/roles/traffic-router/templates/crontab.j2

```diff
@@ -0,0 +1,24 @@
+# File deployed via ansible push
```

Comment:
The license header I believe has to go first here.

Reply:
Double check one of the other places we do this.  I could be wrong on this.

Reply:
Moved license header to the top. That is how it should be.